backend/Tests/Api/GraphQL/__snapshots__/committed-schema.verified.graphql

```diff
@@ -2117,27 +2117,49 @@ enum AccountStatementEntryType {
   TRANSACTION_FEE_REWARD
 }
 
+"Types of account transactions."
 enum AccountTransactionType {
+  "Initialize a smart contract instance."
   INITIALIZE_SMART_CONTRACT_INSTANCE
+  "Update a smart contract instance."
   UPDATE_SMART_CONTRACT_INSTANCE
+  "Transfer CCD from an account to another."
   SIMPLE_TRANSFER
+  "Transfer encrypted amount."
   ENCRYPTED_TRANSFER
+  "Same as transfer but with a memo field."
```

Comment:
Just to be consistent with the other comments.
```suggestion
  "Same as transfer, but with a memo field."
```

Reply:
This is actually generated from an enum in .NET SDK, so I'll need to fix this as a PR for that.